### PR TITLE
Stimpaks (and Supers) make you hungry, and poison you if you're starving.

### PR DESCRIPTION
**Pull request by SeniorScore**
## About The Pull Request

In an effort to encourage forms of healing that just stimpaks without *outright* nerfing their healing ability, I've decided this is a good way to do it (stealing inspiration from some NV mod I can no longer recall).
Really I've just lifted liplocide's effect and just made it deplete nutrition faster, and make more toxins. Removing the toxins healing of stimpaks also helps make the reality of this more damaging.

## Why It's Good For The Game

Make sutures you fucking casuals.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
balance: 
/:cl:

### CODE REVIEW

File: code/modules/fallout/reagents/medicines.dm [hunk 1]
```diff
@@ -41,11 +41,15 @@
 	if(!M.reagents.has_reagent(/datum/reagent/medicine/healing_powder)) // We don't want these healing items to stack, so we only apply the healing if these chems aren't found.We only check for the less powerful chems, so the least powerful one always heals.
 		M.adjustBruteLoss(-4*REAGENTS_EFFECT_MULTIPLIER)
 		M.adjustFireLoss(-4*REAGENTS_EFFECT_MULTIPLIER)
-		M.adjustToxLoss(-1*REAGENTS_EFFECT_MULTIPLIER)
+		//M.adjustToxLoss(-1*REAGENTS_EFFECT_MULTIPLIER)
```
Comment (Looreydev) [inline code/modules/fallout/reagents/medicines.dm:44]:
```suggestion
```

File: code/modules/fallout/reagents/medicines.dm [hunk 2]
```diff
@@ -105,11 +109,15 @@ datum/reagent/medicine/super_stimpak/on_mob_life(mob/living/M)
 	if(!M.reagents.has_reagent(/datum/reagent/medicine/healing_powder/poultice) && !M.reagents.has_reagent(/datum/reagent/medicine/stimpak) && !M.reagents.has_reagent(/datum/reagent/medicine/healing_powder)) // We don't want these healing items to stack, so we only apply the healing if these chems aren't found. We only check for the less powerful chems, so the least powerful one always heals.
 		M.adjustBruteLoss(-8*REAGENTS_EFFECT_MULTIPLIER)
 		M.adjustFireLoss(-8*REAGENTS_EFFECT_MULTIPLIER)
-		M.adjustToxLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
+		//M.adjustToxLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
```
Comment (Looreydev) [inline code/modules/fallout/reagents/medicines.dm:112]:
```suggestion
```

**Comment by Neveralone1744** [comment]
so i may be the dumb but why not just weaken stims? idk if its cause i was a big boy 10 con player but stims never really did that much

**Comment by SeniorScore** [comment]
> so i may be the dumb but why not just weaken stims? idk if its cause i was a big boy 10 con player but stims never really did that much

I wanted to attempt to nerf stims in a way that didn't damage their ability to heal while also demanding more forethought in how/when to use them. And as I also said, to encourage the alternatives to stims, your sutures, ointments and all.

**Comment by Neveralone1744** [comment]
well it is just odd that one super can make you pretty close to immortal to death but i like the additions you have given

**Pull request merged**